lib/ph_model.rb

```diff
@@ -0,0 +1,45 @@
+require 'active_support/all'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.